capstone/src/main/webapp/script.js

```diff
@@ -53,10 +53,10 @@ function createRestaurantElement(restaurant) {
   restaurantElement.appendChild(title);
 
   restaurantElement.addEventListener('click', () => {
-    // Store clicked restaurant in the session
-    sessionStorage.setItem('selectedRestaurant', JSON.stringify(restaurant));
-    // Redirect to restaurant detail page
-    window.location.href = '/restaurantDetails.html';
+    // Redirect to restaurant detail page with correct restaurant key
```

Comment:
Bit of a pet peeve here:

Can this redirect link not just be the HREF on the actual restaurant link in the body, rather than having to go through an event listener that redirects the current page? I'd like to be able to middle-click on 10 links and see them in separate tabs, without the current window redirecting as well. I see too many websites where you can't do a basic tab-browse middle-click because of custom javascript for basic links.

Reply:
I changed it so that it will open in the same window if you just click on a restaurant, but open in a new tab if you ctrl+click, so you can open multiple tabs if desired! Let me know if you meant something else!

Reply:
I think what Tamas was suggesting was wrapping the restaurant name in an anchor tag ("\<a\>") with the href attribute set to the desired url.

The html might look like this:
\<h2\>\<a href="value stored in redirect variable"\>Restaurant Name\</a\>\</h2\>

Using anchor tags over onclick events with redirects is preferred for the reasons Tamas listed, and also because the anchor tags work better with screen readers.

Reply:
Thank you for the clarification! Updated.